src/berry_mill/kiwrap.py

```diff
@@ -85,6 +87,17 @@ def add_repo(self, reponame: str, repodata: Dict[str, str]) -> KiwiParent:
 
         return self
 
+    def _verify_gpg_key(self, key_path) -> bool:
+        """
+        Virify wether the downloaded file is a GPG key
+        """
+        try:
+            return bool(subprocess.run(['gpg', '--import', key_path], capture_output=True, text=True).returncode == 0)
```

Comment:
I would rather not make use of --import, because this manipulates the local gpg keyring. 
I would prefer --dearmor to check for valid OpenPGP data